subsystem/hcp/cluster_resource_test.go

```diff
@@ -825,12 +825,13 @@ var _ = Describe("HCP Cluster", func() {
 					VerifyJQ(`.cloud_provider.id`, "aws"),
 					VerifyJQ(`.region.id`, "us-west-1"),
 					VerifyJQ(`.product.id`, "rosa"),
-
+					VerifyJQ(`.aws.ec2_metadata_http_tokens`, "required"),
 					RespondWithPatchedJSON(http.StatusCreated, template, `[
 					{
 					  "op": "add",
 					  "path": "/aws",
 					  "value": {
+					  	  "ec2_metadata_http_tokens" : "required",
```

Comment:
If we keep it as an optional value could you also include a test for when the value is not supplied that it is being set as optional?

Reply:
@gdbranco 
added test case which no ec2_metadata_http_tokens set, please help to re-review the changes, thanks